bindings/nodejs/examples/offline_signing/3_send_block.ts

```diff
@@ -33,7 +33,7 @@ async function run() {
         const blockId = await onlineClient.blockId(block);
 
         console.log(
-            'Transaction sent: https://explorer.iota.org/devnet/block/' +
+            'Transaction sent: https://explorer.testnet.shimmer.network/testnet/block/' +
```

Comment:
We don't have a .env entry for the explorer ?

Reply:
Updated